api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -491,6 +491,12 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  // The configuration of the extension to use for request ID utilities. This includes operations such as
+  // generation, validation, and associated tracing operations.
+  //
+  // If not set, Envoy uses the :ref:`default UUID-based behavior <extension_envoy.request_id_utils.uuid>`.
+  RequestIDUtils request_id_utils = 36;
```

Comment:
Playing naming poker, I think something like `RequestIdExtension` is probably a better message/field name. The "utils" concept is strictly less descriptive.

Reply:
Sure, can rename it. What's about relevant classes in code, should I keep them as is?